nservicebus/upgrades/5to6.md

```diff
@@ -600,3 +600,11 @@ In Version 6 the callback routing is based on user-provided unique endpoint inst
 snippet: 5to6-Callbacks-InstanceId
 
 NOTE: This identifier needs to be stable and should never be hardcoded, e.g. it can be read from the configuration file or from the environment (e.g. role ID in Azure).
+
+
+## IManageMessageFailures is now Obsolete
+
+In Version 5, the `IManageMessageFailures` interface served as an extension point to customize the process of handling the action when a message fails the first level retries before the message is forwarded to the error queue.  
+
+Version 6 allows more control of this behavior with the [more advanced message processing pipeline](/nservicebus/pipeline/customizing-v6.md) and there is also Version 6 specific guidance for [managing errors with the pipeline](/nservicebus/errors/managing-errors-with-the-pipeline.md).
```

Comment:
i think it is a bit generous to say "Version 6 allows more control of this behavior with the". i dont think v6 is more customizable than v5 in terms of the pipeline. at least not measurable from the consuming point of view